WooCommerce/WooCommerceTests/Mocks/MockABTesting.swift

```diff
@@ -6,15 +6,7 @@ struct MockABTesting {
     /// Sets the provided A/B Test variation in `UserDefaults`, to mock a given experiment assignment
     ///
     static func setVariation(_ variation: AutomatticTracks.Variation, for experiment: ABTest) {
-        let newVariation: String?
-        switch variation {
-        case .control:
-            newVariation = "control"
-        case .treatment(let type):
-            newVariation = type ?? "treatment"
-        }
-
-        let assignment = [experiment.rawValue: newVariation]
+        let assignment = [experiment.rawValue: variation.analyticsValue]
```

Comment:
Do you mind educating me on this particular line here? I couldn't make the connection between "analytic" (I presume it's a text we can query in Sentry?) and "A/B testing assignment".

Reply:
It's been a while and the honest answer is: I don't remember.

Looking at the diff and the codebase, present me can guess past me replaced the custom code with `variation.analyticsValue` because the code here was the same used there:

https://github.com/woocommerce/woocommerce-ios/blob/cdac7d80cdc8f913d996557e9a8f9945566fb094/Experiments/Experiments/ABTest.swift#L60-L70

Your question made me doubt ~myself~ past me's work so I checked how `MockABTesting` is used and discovered that it's unused. So we can just remove it and remove the need to think wether this substitution was appropriate 😄